pyfr/solvers/acnavstokes/system.py

```diff
@@ -12,3 +12,25 @@ class ACNavierStokesSystem(BaseAdvectionDiffusionSystem):
     intinterscls = ACNavierStokesIntInters
     mpiinterscls = ACNavierStokesMPIInters
     bbcinterscls = ACNavierStokesBaseBCInters
+
```

Comment:
Maybe this could be a mixin? `ExternalSystem` or something. You will probably need to come up with a nice syntax for saying what externals are needed though.